### PR TITLE
[Feature] - Allow all locales 

### PR DESCRIPTION
**Pull request by Quentinchampenois**
#### Description 

Allow system administrators to enable all locales in Decidim

- [x] Bump to latest revision
- [x] Update JS depencies 

### How to test

1. Create a new app and seed (seeds will take a moment because it generates translation for each available locale)
2. See available locales in app
3. Create a new organization and select some locales
4. See difference between the two organizations

### CODE REVIEW

File: config/initializers/decidim.rb [hunk 1]
```diff
@@ -1,13 +1,11 @@
 # frozen_string_literal: true
 
 Decidim.configure do |config|
-  config.expire_session_after = ENV.fetch("DECIDIM_SESSION_TIMEOUT", 90).to_i.minutes
```
Comment (paulinebessoles) [inline config/initializers/decidim.rb:4]: Why did you delete this config?

File: config/initializers/decidim.rb [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
   config.application_name = "OSP Agora"
   config.mailer_sender = "OSP Agora <ne-pas-repondre@opensourcepolitics.eu>"
 
   # Change these lines to set your preferred locales
   config.default_locale = :en
-  config.available_locales = [:en, :fr]
```
Comment (paulinebessoles) [inline config/initializers/decidim.rb:10]: We shouldn't add all available locales here?

**Review by paulinebessoles — CHANGES_REQUESTED** [review]

File: config/initializers/decidim.rb (hunk 1 above)
Comment (Quentinchampenois) [reply config/initializers/decidim.rb:4]: Hi, this is a duplicated line, there is the same below

File: config/initializers/decidim.rb (hunk 2 above)
Comment (Quentinchampenois) [reply config/initializers/decidim.rb:10]: I've deleted this line because it takes the default defined in Decidim-core, so we are sure to have all available locales for the given version !

File: config/initializers/decidim.rb (hunk 1 above)
Comment (Quentinchampenois) [reply config/initializers/decidim.rb:4]: But the default is now 180minutes, is it good ?

Comment (paulinebessoles) [reply config/initializers/decidim.rb:4]: Ok thank you for your answer! Yes 180 min is perfect!

**Review by paulinebessoles — APPROVED** [review]

**Review by moustachu — APPROVED** [review]

**Review by armandfardeau — APPROVED** [review]
